### DISCUSSION TITLE
Tempo with pot and LCD or 7segment display?

### DISCUSSION THREAD

**Discussion by interstaller555** [copych/AcidBox#19] [Ideas]
Any chance we can set tempo with a pot and display it on a 7 segment display?

**Comment by copych** [comment]
@interstaller555 sure why not? There's a set_bpm(float newBpm) in AcidBanger.ino right for that purpose.  And line 361 of AcidBox.ino contains commented set_bpm( 40.0f + (paramVal * 160.0f)); . Uncomment it and comment the next line instead. About 7-segment display I can't advice since I don't use ones and don't have any, so you have to test it yourself.
